app/src/docs/Buttons/ButtonMonoInvert.tsx

```diff
@@ -0,0 +1,30 @@
+/* @license Copyright 2023 @paritytech/polkadot-cloud authors & contributors
+SPDX-License-Identifier: GPL-3.0-only */
+
+import { faUser } from "@fortawesome/free-solid-svg-icons";
+import { Button } from "@packages/cloud-react/lib/buttons/Button";
+import { SimpleEditor } from "../lib/SimpleEditor";
+
+export const ButtonMonoInvert = () => {
+  const code = `<Button type="monoInvert" text="Button" marginRight />
+<Button type="monoInvert" text="Button" iconLeft={faUser} marginRight />
+<Button type="monoInvert" text="Button" iconRight={faUser} marginRight />
+<Button type="monoInvert" lg text="Button" disabled />`;
+
+  return (
+    <>
+      <div className="demo">
+        <Button type="monoInvert" text="Button" marginRight />
+        <Button type="monoInvert" text="Button" iconLeft={faUser} marginRight />
+        <Button
+          type="monoInvert"
+          text="Button"
+          iconRight={faUser}
+          marginRight
+        />
+        <Button type="monoInvert" lg text="Button" disabled />
+      </div>
+      <SimpleEditor code={code} />
+    </>
+  );
+};
```

Comment:
In order to reduce repeating and rewriting what we can do is add the of the code in an array and for the `code` variable, join it, while.for the actual code just place ot there as an array. It should be something like:
```suggestion
export const ButtonMonoInvert = () => {
  const code = [
          <Button type="monoInvert" text="Button" marginRight />,
           <Button type="monoInvert" text="Button" iconLeft={faUser} marginRight />, 
           <Button type="monoInvert" text="Button" iconRight={faUser} marginRight />, 
           <Button type="monoInvert" lg text="Button" disabled />
];

  return (
    <>
      <div className="demo">
        {code}
      </div>
      <SimpleEditor code={code.join()} />
    </>
  );
};
```
This way one writes code only once and makes sure that is exactly the same in both actual code and editor

Reply:
Oh yes, in some cases the code is different, such as with the extension icons being wrapped in `svg-box`.